docs/_rtd/development.md

```diff
@@ -50,31 +50,6 @@ Say yes to any prompts and your `conda` environment will be created!
  
 To verify installation, activate your `conda` environment with `conda activate <my_env>`. If you see `(<my_env>)` on the far left of the command prompt, you have successfully created and activated your environment. Type `conda deactivate` to exit at any time.
 
-**Apple Silicon Installation**
-
-`Ark` now supports development with Apple Silicon.
-
-Currently there is not a native M1 implementation of ark-analysis for development, so it will need to go through Rosetta 2 (the Intel to Arm transition layer). Luckily, this isn't something you'll have to deal with, as `conda` makes it straightforward.
-
-1. Create a Python 3.8 `conda` environment called `my_env`.
-    ```sh
-    conda create -n <my_env> python=3.8
-    ```
-
-2. Test to make sure the `platform.machine()` function reports `arm64` in the terminal.
-    ```sh
-    conda activate <my_env>
-    python -c "import platform;print(platform.machine())"
-    ```
-
-3. Force `conda` commands to use Intel Mac packages.
-    ```
-    conda config --env --set subdir osx-64
-    ```
-4. The prompt may ask you to deactivate and reactivate the environment as well.
-
-Now any package that is installed in `my_env` will targeted for `arm64`.
-
 ### Setting up ark-analysis for development
 
 `ark` relies on several other Python packages. Inside the `ark-analysis` repo (if you don't have it, first run `git clone https://github.com/angelolab/ark-analysis.git`), and with your virtual environment activated, you will need to install these other dependencies as well. Run `pip install -r requirements.txt` to do so. 
```

Comment:
This doesn't work anymore, right?

Reply:
It's not needed anymore. The Apple silicon instructions are now the same as with any other OS / architecture.